subscription_manager/common/utils.py

```diff
@@ -119,3 +119,21 @@ def validate_str_field(field: str, none_allowed: bool = False):
         )
     if field == "":
         raise InvalidValueException(EMPTY_FIELD_MSG)
+
+
+def validate_subscription_changes(
+    subscription: Subscription, changes: dict
+) -> Subscription:
+    """
+    Validate changes and return updated Subscription object
+    Args:
+        subscription (Subscription): subscription object to change
+        changes (dict): dict contains subscription changes
+    Returns:
+        Subscription: updated Subscription object to send to to Database
+    """
+    updated_subscription = asdict(subscription)
+    for key, value in changes.items():
```

Comment:
why not updated_subscription.update(changes)?

Reply:
Done